run_unittests.py

```diff
@@ -4324,7 +4416,8 @@ def should_run_cross_mingw_tests():
 
 if __name__ == '__main__':
     unset_envs()
-    cases = ['InternalTests', 'DataTests', 'AllPlatformTests', 'FailureTests', 'PythonTests']
+    cases = ['InternalTests', 'DataTests', 'AllPlatformTests', 'FailureTests',
+            'PythonTests', 'ConfigFileTests']
```

Comment:
__[Flake8]__

[E128] continuation line under-indented for visual indent


<!-- Comment by [Sider](https://sider.review) -->